proposals/raster_to_vector.json

```
@@ -0,0 +1,53 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts a raster to a vector",
```

I don't think "a raster" and "a vector" is proper terminology. I would at least say 
```suggestion
    "summary": "Converts raster data to vector data",
```